api/src/controllers/listing.controller.ts

```diff
@@ -89,20 +89,18 @@ export class ListingController {
   }
 
   // REMOVE_WHEN_EXTERNAL_NOT_NEEDED
-  @Get('combined')
+  @Post('combined')
```

Comment:
This was changed to a post because we have a new param, a list of listing IDs, that we pass to this function to filter by the visible listings. In a zoomed out list, this was too large to be passed in a header.